ui/src/services/utils.js

```diff
@@ -329,3 +329,26 @@ export const fromUnixTimestampToDate = (date) => {
     return new Date(date * 1000);
   }
 };
+
+// Convert the timestamp in milliseconds to an object with day, hour, minute and second
+export const fromMilliSectoAge = (milliSecTime) => {
+  if (milliSecTime && milliSecTime > 0) {
+    let day, hour, minute, second;
+
+    second = Math.floor(milliSecTime / 1000);
+    minute = Math.floor(second / 60);
+    hour = Math.floor(minute / 60);
+    day = Math.floor(hour / 24);
+
+    second = second % 60;
+    minute = minute % 60;
+    hour = hour % 24;
+
+    return {
+      day: day,
+      hour: hour,
+      minute: minute,
+      second: second,
+    };
+  }
```

Comment:
What happen when `milliSecTime` is `null` or `undefined` ?
Do you chose to return `undefined` or you want to return something like this ?
```javascript
{
      day: 0,
      hour: 0,
      minute: 0,
      second: 0,
};
```
Or something else ?